src/components/forms/validations.jsx

```diff
@@ -44,5 +45,5 @@ module.exports.validationHOCFactory = function (defaultValidationErrors) {
 };
 
 module.exports.defaultValidationHOC = module.exports.validationHOCFactory({
-    isDefaultRequiredValue: 'This field is required'
+    isDefaultRequiredValue: <intl.FormattedMessage id="teacherRegistration.validationRequired" />
```

Comment:
I would say having it in `forms` constitutes it being outside of `teacherregistration`.